src/services/apimService.ts

```diff
@@ -252,7 +252,73 @@ export class ApimService extends BaseService {
       });
     } catch (e) {
       this.log("Error creating APIM Property");
+      this.log(JSON.stringify(e.body, null, 4));
       throw e;
     }
   }
+
+  /**
+   * Creates the XML payload that defines the API operation policy to link to the configured backend
+   */
+  private createApiOperationXmlPolicy(): string {
```

Comment:
should we have tests to verify this object data format?

Reply:
This would need to be an e2e integration test to the azure API, otherwise I've validated the shape of the xml document.